dbeaver/dbeaver#37811 Update the GaussDB driver jar package and corresponding configuration template

MuStevenic opened the pull request:
Our original commit #39162 encountered the “org.jkiss.dbeaver.ext.postgresql.test 1.0.120-SNAPSHOT FAILURE” issue, which we could not reproduce locally.
So we split the original commit into a number of small feature commits.
This commit is the first one, which resolves the following problem in issue #37811: "Secondly, the GaussDB driver jar package provided by Huawei Cloud is outdated and needs to be updated to gaussdbjdbc - 506.0.0.b058.jar."

MuStevenic commented:
Hi admin, here my commit message does not meet the specification and does not pass the test.So,do I need to reopen a new pull request and submit a message that meets the format, or do I need to follow the tutorials provided in the Commit Message Validation about the github community to change the old commit message and force it to be pushed?
@labanovichttps  I would appreciate it if you could be available to answer my questions!

arhayka commented:
Hi @MuStevenic! Thank you for the contribution! I will pass this PR to our product team for review and testing.

MuStevenic commented:
<img width="1895" height="756" alt="image" src="https://github.com/user-attachments/assets/3e077072-bf7e-486f-896e-9708f12833dc" />

@arhayka @labanovichttps 
Hello admin, I would like to inquire about the cause of this commit error. I have re-uploaded my code and the commit message as per the tutorial provided in the checkout error, and currently my commit message looks like this: dbeaver/dbeaver#37811 Update the GaussDB driver jar package and corresponding configuration template. 
This is what I have changed after mimicking someone else's commit message, and as I understand it, it looks like it already complies with the rules in the error message, but this commit still doesn't pass the check.
So please tell me how should I modify my commit message.
I would be grateful if you could answer my question!

labanovichttps commented:
> This is what I have changed after mimicking someone else's commit message, and as I understand it, it looks like it already complies with the rules in the error message, but this commit still doesn't pass the check.

Could you please edit this message as `dbeaver/dbeaver#37811` or  just put `#37811`  .Thank you!

MuStevenic commented:
> Could you please edit this message as or just put .Thank you!`dbeaver/dbeaver#37811``#37811`
@labanovichttps Sorry, I would like to understand further what you mean by that.
Do I just need to change the commit message or do I need to resubmit a new pull request?
I apologize for the trouble and look forward to your guidance!

MuStevenic commented:
> Could you please edit this message as or just put .Thank you!`dbeaver/dbeaver#37811``#37811`
@labanovichttps Sorry, I would like to understand further what you mean by that.
Do I just need to change the commit message or do I need to resubmit a new pull request?
I apologize for the trouble and look forward to your guidance!

MuStevenic committed:
#37811 Update the GaussDB driver jar package and corresponding configuration template

#37811 Update the GaussDB driver jar package and corresponding configuration template

MuStevenic commented:
@labanovichttps Hello Administrator!
I've modified my commit message for the time being, could you please check the code again when you have time so that I can observe whether it passes the check or not, thanks!

MuStevenic commented:
@labanovichttps Hello Administrator!
I've modified my commit message for the time being, could you please check the code again when you have time so that I can observe whether it passes the check or not, thanks!

labanovichttps commented:
> @labanovichttps Hello Administrator! I've modified my commit message for the time being, could you please check the code again when you have time so that I can observe whether it passes the check or not, thanks!

@MuStevenic Our piepline checks all the messages of pr, could u make new pr please with correct messages? or git rebase -i ?

MuStevenic commented:
@labanovichttps Hi Admin,I've opened a new pr recently,if you have time please help me to check the code again,thanks a lot!
The new pr links are as follows：https://github.com/dbeaver/dbeaver/pull/39781
